app/components/exp-video-config-quality/component.js

```diff
@@ -438,7 +460,61 @@ export default ExpFrameBaseComponent.extend(VideoRecord, {
             this.set('showWarning', false);
             this.set('hasPlayedBack', false);
             this.destroyRecorder();
-            this.setupRecorder(this.$(this.get('recorderElement')));
+            document.getElementById('recordingIndicator').classList.remove('text-danger','Blink');
+            this.setupRecorderAndUpdateDisplay();
+        },
+
+        playvideo() {
+            document.getElementById('recordingIndicator').classList.remove('text-danger','Blink');
+            const vid = document.querySelector('video');
+            vid.src = null;
+            vid.srcObject = null;
+            vid.muted = false;
+            vid.volume = 1;
```

Comment:
I am seeing this in two places.  Should this be de-duplicated?

Reply:
Great catch - thanks!! I've refactored this to remove the duplication.